SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -77,6 +78,14 @@ internal class DeploymentLauncher
 
             return confirmUploadResponse.Snapshot.Id;
         }
+        private static PlatformApiEndpoint GetApiEndpoint(string region)
```

Comment:
```suggestion
        
        private static PlatformApiEndpoint GetApiEndpoint(string Region)
```

Reply:
This C# project seems to use a different coding standard, should we stick with it or start transitioning to the UE4 one?